### ISSUE TITLE
1.15 release

### ISSUE THREAD

**Issue by blairconrad** [FakeItEasy/FakeItEasy#186]
**Ready** when all other issues forming part of the release are **Done**.
- [x] run code analysis in VS and address violations (send a regular PR which must be merged before continuing)
- [x] use `rake set_version[new_version]` to change SharedAssemblyInfo.cs to stable version (of form _xx.yy.zz_)
- [x] nuspec - update releaseNotes, mentioning non-owner contributors, if any
- [x] push to origin branch, create PR to upstream master
- [x] check build, merge PR
- [x] check build, create release in [GitHub UI](https://github.com/FakeItEasy/FakeItEasy/releases) (copy release notes from nuspec to description and attach nuget package)
- [x] push nuget package
- [x] de-list pre-release nuget packages if present
- [x] update website with contributors list (if in place)
- [x] tweet, mentioning contributors and post link as comment here for easy retweeting ;-)
- [x] post tweet in JabbR ([fakeiteasy](https://jabbr.net/#/rooms/fakeiteasy) and [general-chat](https://jabbr.net/#/rooms/general-chat))
- [x] change SharedAssemblyInfo.cs to next pre-release version (of form _xx.yy.zz-beta01_)
- [x] nuspec - change release notes to TBD
- [x] push to origin branch, create PR to upstream master
- [x] check build, merge PR
- [x] create a new milestone for the next release
- [x] create new issue based on this one for next release, adding it to the new milestone
- [x] close all issues on this milestone
- [x] close this milestone
